app/views/users/_toolbar.html

```diff
@@ -0,0 +1,31 @@
+<div class="toolbar-container">
+  <div id="toolbar">
+    <div class="title-container">
+      <h2><%= link_to l(:label_user_plural), :controller => '/users', :action => 'index' %> &#187; <%= new_user ? l("label_user_new") : h @user.name %></h2>
+    </div>
+    <% unless new_user %>
+      <ul id="toolbar-items">
+        <li class="toolbar-item">
+          <%= link_to user_path(@user), class: 'button' do %>
+            <i class="button--icon icon-user1"></i> <%= l(:label_profile) %>
+          <% end %>
+        </li>
+        <% unless current_user.id == @user.id %>
+          <%= form_for @user, :url => {:action => :change_status},
+                    :method => :post do %>
+            <li class="toolbar-item">
+              <%= change_user_status_buttons(@user) %>
+            </li>
+          <% end %>
+        <% end %>
+        <% if Setting.users_deletable_by_admins? %>
```

Comment:
I am not sure yet where this component is used, but what about deleting ones own account? There is a separate setting for that...